### PR TITLE
Fixed incorrect negative search on multi attribute values

### PR DESCRIPTION
**Pull request by Toflar**
Alternative to https://github.com/loupe-php/loupe/pull/19

### CODE REVIEW

File: src/Internal/Filter/Ast/Operator.php [hunk 1]
```diff
@@ -14,9 +15,10 @@ enum Operator: string
     case In = 'IN';
     case LowerThan = '<';
     case LowerThanOrEquals = '<=';
+    case NotEquals = '!=';
```
Comment (alexander-schranz) [inline src/Internal/Filter/Ast/Operator.php:18]: should this not be in an own pull request 😄

Comment (Toflar) [reply src/Internal/Filter/Ast/Operator.php:18]: Yeah I wanted to have all the opposites avaiable in this PR. It still needs to be supported in the parser though.
